packages/core/src/components/ozon-content/nodes/figuur.node.tsx

```diff
@@ -53,6 +60,9 @@ export class OzonContentFiguurNode implements OzonContentNode {
     const illustratieNode = childNodes.find((n) => getNodeName(n) === "Illustratie");
     const bijschriftNode = childNodes.find((n) => getNodeName(n) === "Bijschrift");
 
+    const editAction = node.getAttribute("wijzigactie") as FiguurEditAction;
```

Comment:
In principe is `as` een slecht idee. Met de type assertion zeg jij dat je het beter weet dan de TypeScript compiler. In deze situatie zit je relatief veilig omdat de STOP standaard zo potdicht zit, maar kijk eens naar de verschillen van het `wijzigactie` attribuut in 1.3.0 vs. 2.0.0-rc:

* https://koop.gitlab.io/STOP/standaard/1.3.0/tekst_xsd_Attribute_Group_tekst_agWijzigactie.html
* https://koop.gitlab.io/STOP/standaard/2.0.0-rc/tekst_xsd_Attribute_Group_tekst_agWijzigactie.html

Beter zou zijn om hier een functie voor te schrijven die de waardes test alvorens de TypeScript compiler te corrigeren. Bv met user defined type guards of assertion functions.